aspnet/fundamentals/localization.rst

```diff
@@ -203,7 +203,16 @@ If you only pass in one of the two (``culture`` or ``ui-culture``), the query st
 CookieRequestCultureProvider
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie. The sample app provides code to set the cookie for French and English cultures.
+In old versions of ASP.NET a production apps will often provide a mechanism to set the culture with the ``ASPNET_CULTURE`` cookie.
```

Comment:
So then I would suggest "ASP.NET Core applications can use cookies to configure culture settings with ``.AspNetCore.Culture`` as the default cookie name. You have to ..."